tests/e2e/specs/wordpress/adminMenu.js

```diff
@@ -48,7 +48,9 @@ describe('Admin Menu', () => {
       visible: true,
     });
 
-    await percySnapshot(page, 'Admin Menu');
+    await percySnapshot(page, 'Admin Menu', {
+      percyCSS: `.postbox { display: none; }`,
```

Comment:
Let's also hide update bubbles and the WordPress version number.

* `span.update-plugins { display: none !important; }` (yes, needs `!important`)
* `#footer-upgrade { display: none; }`

Reply:
See https://github.com/google/web-stories-wp/pull/4979#discussion_r512632854